tests/formdesigner.ignoreButRetain.js

```diff
@@ -2,11 +2,17 @@
 define([
     'tests/utils',
     'chai',
-    'jquery'
+    'jquery',
+    'text!static/ignoreButRetain/empty-parent.xml',
+    'text!static/ignoreButRetain/renamed.xml',
+    'text!static/ignoreButRetain/unrenamed.xml'
```

Comment:
No new tests in this file, just moved XML out to files. Makes debugging test failures much easier.